examples/global-cluster-config/zonemapping.json

```diff
@@ -0,0 +1,57 @@
+{
+   "AWSTemplateFormatVersion":"2010-09-09",
+   "Description":"This template creates a cloud provider snapshot restore job on the MongoDB Atlas API, this will be billed to your Atlas account.",
```

Comment:
```suggestion
   "Description":"This template creates a Global Cluster Config resource on the MongoDB Atlas API, this will be billed to your Atlas account.",
```